### PR TITLE
Added subdirectory input validation

### PR DESCRIPTION
**Pull request by tajpouria**
Validate module subdirectory before getting `{"success":false,"info":"provided sub directory is not valid as it does not end with a /"}
` error on webhooks!
By the way with this implementation input likes '//' are valid. It just makes sure that either input does not provide or ends with '/' and I don't know if that's fine or not.

### CODE REVIEW

**Comment by lucacasonato** [comment]
Thanks! I hadn't gotten around to this before release. I think the validation is actually that it may not start with a /, but must end in a slash. Could you update that in the validation regex?

**Comment by tajpouria** [comment]
@lucacasonato Here you go!

**Comment by lucacasonato** [comment]
@tajpouria Thanks, could you fix the lint error CI is complaining about? Then this is good to merge.

**Comment by tajpouria** [comment]
@lucacasonato My bad!

File: components/RegistryInstructions.tsx [hunk 1]
```diff
@@ -207,12 +214,24 @@ function RegistryInstructions(props: { isOpen: boolean; close: () => void }) {
...
+                            }`}
                             type="text"
                             placeholder="Subdirectory"
                             value={subdirectory}
                             onChange={(e) => setSubdirectory(e.target.value)}
                           />
+                          {isSubdirectoryValid === false ? (
+                            <p className="text-red-400 mb-2">
+                              Provided sub directory is not valid it should not
+                              start with a / and should end with a /
```
Comment (lucacasonato) [inline components/RegistryInstructions.tsx:232]: Change to:

`The provided subdirectory is not valid. It must end with a <InlineCode>/</InlineCode>, but may not start with one. (e.g. <InlineCode>src/</InlineCode>)`

Comment (tajpouria) [reply components/RegistryInstructions.tsx]: Changed

**Pull request merged**